spec/models/item_spec.rb

```diff
@@ -434,6 +434,22 @@
     end
   end
 
+  describe "when distribution_quantity is set by default" do
+    it "should set distribution_quantity to 50 for regular items" do
+      item = Item.new
+      item.valid?
```

Comment:
You shouldn't need to call this - your callback is `after_initialize`, not `before_validate`.